src/ess/livedata/config/instruments/bifrost.py

```diff
@@ -232,8 +257,11 @@ def _spectrum_view(params: BifrostWorkflowParams) -> StreamProcessorWorkflow:
     return StreamProcessorWorkflow(
         wf,
         dynamic_keys={'unified_detector': NeXusData[NXdetector, SampleRun]},
-        target_keys=(SpectrumView,),
-        accumulators=(SpectrumView,),
+        target_keys=(SpectrumView, AnalyzerCounts, BankCounts, DetectorCounts),
```

Comment:
Do we need all three extra counts? Or is it wasteful to compute and stream all 3? 

You could stream only `AnalyzerCounts` and sum in the frontend.